{
  try {
    callback.setDone(transform.fun(result));
  }
 catch (  Throwable e) {
    CommandProcessor.LOG.error(e);
  }
}
